lib/api/src/grpc/proto/points.proto

```diff
@@ -117,6 +117,11 @@ message SearchPoints {
   optional uint64 offset = 9; // Offset of the result
 }
 
+message SearchBatchPoints {
+  string collection_name = 1; // Name of the collection
```

Comment:
The name of the collection is passed explicitly although the `search_points` contain a dedicated `collection_name` field.

The service validates that the `collection_name` exists and assumes all points in the batch targets the same collection.

This is not extremely clean for the time being.